assets/css/styles.css

```diff
@@ -89,3 +91,158 @@ ul {
 .social {
   width: 15px;
 }
+
+.recent-works-title {
+  background: url("../../assets/img/Indicator.png") 0 0 no-repeat;
+  background-position: bottom center;
+}
+
+.scroll-down {
+  background-color: rgba(0, 0, 0, 0);
+  border-style: hidden;
+  margin: 3em 0 0;
+}
+
+.disabled-button:hover {
+  background-color: #1e488f;
+}
+
+.works-wrapper {
+  display: grid;
+  grid-template-rows: 474px 474px 474px 474px 474px 474px;
+  gap: 2em;
+  margin: 3em 0 1em 0;
+}
+
+.works-card {
+  background-size: 100%;
+  flex-direction: column;
+  justify-content: flex-end;
+  border-radius: 8px;
+  border: 1px solid #d0d9d7;
+}
+
+.project1 {
+  background: url("../../assets/img/1.png") 0 0 no-repeat;
+}
+
+.project2 {
+  background: url("../../assets/img/2.png") 0 0 no-repeat;
+}
+
+.project3 {
+  background: url("../../assets/img/3.png") 0 0 no-repeat;
+}
+
+.project4 {
+  background: url("../../assets/img/4.png") 0 0 no-repeat;
+}
+
+.project5 {
+  background: url("../../assets/img/5.png") 0 0 no-repeat;
+}
+
+.project6 {
+  background: url("../../assets/img/6.png") 0 0 no-repeat;
+}
```

Comment:
- [ ] **[OPTIONAL]** To avoid repeat the same code for many classes qe could use `.project1, .project2, .project3, ...{(_the rest of the code_)}` 😃

Reply:
Hello @franclobo , thanks for the comment 👍 . But the code is different for each project class. I am assigning different background images to each class.
Please assist with an illustration if there is a way I can do this.

Reply:
Hello @JonahKayizzi, I'm sorry my fault, You have all the reasons those are images and it is not possible to group them. 😞 I am going to be more careful next time. 👍🏽

Reply:
Alright.
Thanks for the reviews. Let me add the margin and submit 👍